### PR TITLE
Setting up viem-cli tool module

### PR DESCRIPTION
**Pull request by noateden**
Hi, I have set up a module for viem-cli that may be helpful for development. I'm integrating viem modules into viem-cli command, like:

```bash
viem-cli getBlock --chain=ethereum --number=17527200
```

---

## PR-Codex overview
This PR adds new CLI commands to interact with various blockchain networks. It also includes improvements to the CLI interface and adds new dependencies.

### Detailed summary:
- Added new CLI commands: `getChainId`, `getGasPrice`, `getBlockNumber`, `getTransaction`, `getTransactionReceipt`, `getBlock`, `getBlockTransactionCount`, `getBalance`
- Added new dependencies: `bignumber.js`, `isomorphic-ws`, `yargs`
- Improved CLI interface with better argument handling and output formatting.

> The following files were skipped due to too many changes: `src/cli/command.ts`, `src/cli/modules/logs/getLogs.ts`, `src/cli/index.ts`, `pnpm-lock.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by noateden** [comment]
Added some basic and public commands:
- getChainId
- getGasPrice
- getBalance
- getTransactionCount
- getBlock
- getBlockNumber
- getBlockTransactionCount
- getTransaction
- getTransactionreceipt
- getLogs

File: package.json
Comment (izayl) [inline package.json:1]: cli should be a individual package, please not install deps on workspace root.
see more about workspace: https://pnpm.io/workspaces

Comment (noateden) [reply package.json:1]: Hey @izayl, Do you have any setup suggestions for me? I will maintain it for sure!

**Comment by tmm** [comment]
@noateden super cool to see this work! Since this is outside of the scope of what we want to maintain right now, want to publish this as separate package and you can post about it in the [discussions area](https://github.com/wagmi-dev/viem/discussions)? (If you are new to package publishing, [tsup](https://tsup.egoist.dev/#building-cli-app) makes it really easy to build a CLI tool. Feel free to [DM me on Twitter](https://twitter.com/awkweb) if you need any help.)

**Pull request closed without merging**